1.12/Dockerfile.fedora

```diff
@@ -58,10 +60,13 @@ COPY ./root/ /
 RUN sed -i -f /opt/app-root/nginxconf-fed.sed /etc/nginx/nginx.conf && \
     mkdir -p /opt/app-root/etc/nginx.d/ && \
     mkdir -p /opt/app-root/etc/nginx.default.d/ && \
+    mkdir -p ${NGINX_CONTAINER_SCRIPTS_PATH}/nginx-start && \
     chmod -R a+rwx /opt/app-root/etc && \
     chmod -R a+rwx /var /run && \
+    chmod -R a+rwx ${NGINX_CONTAINER_SCRIPTS_PATH}/nginx-start && \
```

Comment:
Why 'a+rwx'? Permissions are too loosen.

Reply:
I just copied from different line w/o thinking too much about it.

Reply:
Container should be able to run with default user and with randum UID+root group. `nginx-start` is owned by root group, ug+rwx should be enought there...